### PR TITLE
PD3 Jakub Lis

### PR DESCRIPTION
**Pull request by liskuba**

### CODE REVIEW

**Comment by ptomaszewska** [comment]
Dobrze wykonana praca domowa 
- dobrze, że podano referencję dot. ablation study (na przyszłość lepiej tworzyć odnośnik do bardziej naukowych źródeł) i źródła kodu
- raczej nie porównuje się wielkości loss między różnymi architekturami bo różna złożoność modeli powoduje że startowa strata jest różna a więc nie do końca sprawiedliwym jest porównywanie tylko końcowej wartości (lepiej opierać się na miarach jakości)
- jeśli dwie architektury mają zbliżone miary jakości to należy wybrać tą o mniejszej liczbie parametrów (warto na tak wyrażony rozmiar sieci zwracać uwagę)
- architektury sieci neuronowych warto byłoby zwizualizować w postaci "bloczków" - jest do tego gotowa funkcja w kerasie
zabrakło zbiorczej tabelki na końcu oraz bardziej pogłębionych wniosków
- aby zdiagnozować czy pojawia się overfitting warto spojrzeć na wykres loss
- szkoda, że nie udało się zdefiniować eksperymentu, w ramach którego accuracy byłoby dużo gorsze niż bazowe
